### PR TITLE
Created staff model, controller, views

### PR DESCRIPTION
**Pull request by jhwebbjr**

### CODE REVIEW

File: app/models/staff_member.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  # end
+  delegate :email=, to: :user
+  # def password=(new_password)
+  #   user.password = new_password
+  # end
+  delegate :password=, to: :user
+  # def password_confirmation=(new_password)
+  #   user.password_confirmation = new_password
+  # end
+  delegate :password_confirmation=, to: :user
```
Comment (Rdore88) [inline app/models/staff_member.rb]: Should we keep those lines that are commented out?

File: config/routes.rb [hunk 2]
```diff
@@ -4,11 +4,17 @@
...
   resources :orders, only: [:create]
   resources :line_items
   resources :products, only: [:index, :show]
   resource :users, only: [:new, :create]
 
+  # namespace :staff_members do
+  #   resources :orders, only: [:show]
+  #   resources :products
+  # end
+
```
Comment (Rdore88) [inline config/routes.rb]: Should we keep the commented out lines?

File: db/schema.rb [hunk 3]
```diff
@@ -59,6 +59,22 @@
...
+  end
+
+  create_table "staffs", force: :cascade do |t|
+    t.string   "name"
+    t.integer  "user_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_staffs_on_user_id", using: :btree
+  end
+
```
Comment (Rdore88) [inline db/schema.rb]: Do we need both of these tables?

File: app/models/staff_member.rb (hunk 1 above)
Comment (jhwebbjr) [reply app/models/staff_member.rb]: We can delete them

File: config/routes.rb (hunk 2 above)
Comment (jhwebbjr) [reply config/routes.rb]: No

File: db/schema.rb (hunk 3 above)
Comment (jhwebbjr) [reply db/schema.rb]: Only the staff_member table is needed. Delete staffs

File: db/seeds.rb [hunk 4]
```diff
@@ -5,3 +5,10 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
```
Comment (Rdore88) [inline db/seeds.rb]: Do we need these?

**Review by Rdore88 — APPROVED** [review]

**Pull request merged**